demo/b2share_demo/config.py

```diff
@@ -21,42 +21,82 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""Demonstration configuration."""
+"""Demonstration configuration, to be changed by the site administrator"""
```

Comment:
Does it mean that the demo can be used production? This module was intended to be used only for demonstration purpose. Normally one shouldn't try to modify the config.py files. The two ways for users to modify the config are:
- the <instance_directory>/b2share.cfg file.
- environment variables.